dark/index.html

```diff
@@ -190,25 +190,7 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
 
     <!-- Contact Section -->
     <div id="contact">
-      <h1>CONTACT ME</h1>
-      <h3>CONTACT <span class="text-info">ME</span></h3>
-
       <div class="formContainer bg-dark p-5 rounded-3">

```

Comment:
Can I know why did you keep these lines? @PiyushXCoder

Reply:
I thought that also needed to be removed. I have reverted.

Reply:
No no you have misunderstand it, I asked why did you keep? I didn't ask why did you remove them??? You have to remove all the formContainers

Reply:
So, Should I remove Those again?